assets/style/style.scss

```diff
@@ -0,0 +1,4 @@
+@import '_core';
+@import '_header';
+@import '_main';
+@import '_footer';
```

Comment:
```suggestion
@import '_footer';
```

Reply:
Ako se slazes sa komentarom samo klikni iznad "Commit suggestion" (Ili "Add suggestion to batch" ako ima vise kometara za prihvatanja a da svi budu zajedno komitovani na git)